packages/grid/_modules_/grid/hooks/features/columns/useColumns.ts

```diff
@@ -115,7 +149,7 @@ const getUpdatedColumnState = (
 export function useColumns(columns: Columns, apiRef: ApiRef): InternalColumns {
   const logger = useLogger('useColumns');
   const [gridState, setGridState, forceUpdate] = useGridState(apiRef);
-
+  const viewportWidth = gridState.containerSizes ? gridState.containerSizes.viewportSize.width : 0;
```

Comment:
I think that it would be better to keep the same name for variables, `containerWidth` sounds better